terasoluna-gfw-dependencies/terasoluna-gfw-dependencies-web-jsp/pom.xml

```diff
@@ -0,0 +1,31 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>terasoluna-gfw-dependencies</artifactId>
+        <groupId>org.terasoluna.gfw</groupId>
+        <version>5.2.0-SNAPSHOT</version>
+        <relativePath>../../terasoluna-gfw-dependencies/pom.xml</relativePath>
+    </parent>
+    <artifactId>terasoluna-gfw-dependencies-web-jsp</artifactId>
+    <packaging>pom</packaging>
+
+    <name>TERASOLUNA Server Framework for Java (5.x) for web-jsp library</name>
+    <url>http://terasoluna.org</url>
+
+    <dependencies>
+        <!-- == Begin TERASOLUNA == -->
+        <dependency>
+            <groupId>${project.groupId}</groupId>
+            <artifactId>terasoluna-gfw-web-jsp</artifactId>
+        </dependency>
+        <!-- == End TERASOLUNA == -->
+        <!-- == Begin Other JavaEE/SE APIs == -->
+        <dependency>
+            <groupId>org.apache.taglibs</groupId>
+            <artifactId>taglibs-standard-jstlel</artifactId>
+        </dependency>
```

Comment:
- [ ] I think this dependency is provided scope and optional, because it should take only if necessary.

terasoluna-gfw : https://github.com/terasolunaorg/terasoluna-gfw/blob/master/terasoluna-gfw-web-jsp/pom.xml#L94
terasoluna-gfw-web-multi-blank : https://github.com/terasolunaorg/terasoluna-gfw-web-multi-blank/blob/master/projectName-web/pom.xml#L103